src/components/VmStatus/VmStatus.js

```diff
@@ -127,13 +181,9 @@ export const VmStatus = ({ vm, launcherPod, importerPods, migration, verbose })
       return <StateVmiWaiting linkTo={getSubPagePath(vm, VirtualMachineModel, 'events')} />;
     case VM_STATUS_STARTING:
       return <StateStarting linkTo={getSubPagePath(launcherPod, PodModel, 'events')} message={statusDetail.message} />;
-    case VM_STATUS_IMPORTING:
-      return (
-        <StateImporting
-          linkTo={getSubPagePath(statusDetail.pod, PodModel, 'events')}
-          additionalText={verbose ? getAdditionalImportText(statusDetail.pod) : ''}
-        />
-      );
+    case VM_STATUS_PREPARING_DISKS:
+    case VM_STATUS_DISKS_FAILED:
+      return <StateDisks vm={vm} statusDetail={statusDetail} verbose={verbose} />;
```

Comment:
IMO, having two components would be better, it gets switched two times, here and inside the component (with the same meaning)

Reply:
is this resolved by making component out of showVmStatus ? or do you mean something else ?

Reply:
Ok, thanks, I get it. I split those two cases to two separate components